bootstrap/src/lib/server.ts

```diff
@@ -49,8 +54,18 @@ export const initHandler = (
   })
 
   app.listen(port, () => logger.info(`Listening on port ${port}!`))
-
   process.on('SIGINT', () => {
     process.exit()
   })
 }
+
+function setupMetricsServer() {
+  const metricsApp = express()
+  const metricsPort = process.env.METRICS_PORT || 9080
+
+  metricsApp.get('/metrics', (_, res) => {
+    res.send(client.register.metrics())
```

Comment:
I think this returns a Promise. `res.send(await client.register.metrics())` should do it

Reply:
Nice catch!